src/content/docs/deployment/variants/binaries.mdx

```diff
@@ -27,23 +27,21 @@ support for it included.
 #### Runtime requirements
 
 For Linux:
-- FFmpeg 6
+- FFmpeg 6.0 or higher, version 6.1 recommended.
```

Comment:
recomended is a bad choice of word, the specific version is required, but we will download if it is missing. I don't think this explains the situation well